rust-toolchain.toml

```diff
@@ -1,2 +1,2 @@
 [toolchain]
-channel = "1.77.2"
+channel = "1.79.0"
```

Comment:
Just linking this here (but you would have seen it on the other PR), so feel free to resolve this comment when you read it.

https://github.com/stackabletech/operator-templating/pull/406

Reply:
Yes, was going to look at the errors today :+1: and thanks for the rust-toolchain PR